sdks/java/testing/expansion-service/build.gradle

```
@@ -27,6 +27,7 @@ ext.summary = """Testing Expansion Service used for executing cross-language tra
 
 
 dependencies {
+  implementation project(":sdks:java:extensions:google-cloud-platform-core")
```

```suggestion
  implementation project(":sdks:java:extensions:google-cloud-platform-core")
  permitUnusedDeclared ":sdks:java:extensions:google-cloud-platform-core"
```

Not 100% sure if this is the right way to do this, but we need something to satisfy the dependency analyzer in Java Precommit

---

Actually, just declaring this as a "testRuntimeOnly" dependency is adequate to include the classes in the shaded jar.,